backend/src/main/kotlin/com/group7/artshare/service/EventService.kt

```diff
@@ -0,0 +1,88 @@
+package com.group7.artshare.service
+
+import com.group7.artshare.entity.*
+import com.group7.artshare.repository.ArtItemRepository
+import com.group7.artshare.repository.ImageRepository
+import com.group7.artshare.repository.OnlineGalleryRepository
+import com.group7.artshare.repository.PhysicalExhibitionRepository
+import com.group7.artshare.request.OnlineGalleryRequest
+import com.group7.artshare.request.PhysicalExhibitionRequest
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.web.server.ResponseStatusException
+
+@Service
+class EventService(
+    private val physicalExhibitionRepository: PhysicalExhibitionRepository,
+    private val onlineGalleryRepository: OnlineGalleryRepository,
+    private val artItemRepository: ArtItemRepository,
+    private val imageRepository: ImageRepository
+) {
+    fun createPhysicalExhibition(
+        physicalExhibitionRequest: PhysicalExhibitionRequest,
+        user: RegisteredUser
+    ): PhysicalExhibition {
+        val newPhysicalExhibition = PhysicalExhibition()
+        if (user is Artist) {
+            newPhysicalExhibition.creator = user
+            user.hostedEvents.add(newPhysicalExhibition)
+        } else
+            throw ResponseStatusException(HttpStatus.BAD_REQUEST, "Registered users cannot create physical exhibitions")
+        newPhysicalExhibition.location = physicalExhibitionRequest.location
+        newPhysicalExhibition.rules = physicalExhibitionRequest.rules
+        if (physicalExhibitionRequest.eventInfo?.posterId?.let { imageRepository.existsById(it) } == false)
```

Comment:
throw error in the beginning. 
This can cause error since we create and related event to the user.